src/zenml/config/server_config.py

```diff
@@ -257,6 +260,10 @@ class ServerConfiguration(BaseModel):
     device_expiration_minutes: Optional[int] = None
     trusted_device_expiration_minutes: Optional[int] = None
 
+    generic_api_token_lifetime: int = (
```

Comment:
We can use `PositiveInt` from `pydantic` to do this validation for us automatically (or `NonNegativeInt` in case we want to allow 0 seconds?)